pom.xml

```diff
@@ -257,6 +257,7 @@
                         <failOnError>false</failOnError>
                         <additionalparam>${javadoc.options}</additionalparam>
```

Comment:
Should this line be removed as well?  Since we're removing the javadoc.options property below, should we just remove this reference as well to avoid confusion?